Tests/bin/MarlinTests.sh

```diff
@@ -10,7 +10,7 @@
 #----- parse command line - first argument is the
 #      test to run
 ## Drop existing ClicPerformance from MARLIN_DLL
-MARLIN_DLL=`echo $MARLIN_DLL | sed -e 's/:\?[-a-z0-9A-Z_/.]*libClicPerformance.so[.0-9]*:\?/:/'`
```

Comment:
Please keep the potential so-version at the end. Just because there is no problem in key4hep, doesn't mean there might not be a problem somewhere else, or at some other time.

Reply:
Ok, I added them again.